component-maps.yml

```diff
@@ -292,20 +293,29 @@ docker_image:
   mender-client-docker-addons:
     git:
     - integration
+    - mender
+    - monitor-client
```

Comment:
monitor-client is not part of mender-client-docker-addons (at least at the moment).

Reply:
Why?

Reply:
I don't know. Maybe because it's a closed component? I just looked at what it currently contains.